packages/isomorphic-lib/src/types.ts

```diff
@@ -1227,6 +1270,8 @@ export const GetUsersResponse = Type.Object({
 
 export type GetUsersResponse = Static<typeof GetUsersResponse>;
 
+export const GetBroadcastsRequest = Type.Object({});
```

Comment:
Why are you defining this as an empty object?

Reply:
Apologies. Removed that now.